tests/php/CampaignAdminTest.php

```diff
@@ -129,4 +134,72 @@ public function testFilters()
             $results
         );
     }
+
+    public function testReadCampaignResponse()
+    {
+        // Login as admin user
+        $group = $this->objFromFixture(Group::class, 'admins');
+        Permission::grant($group->ID, 'ADMIN');
+
+        $admin = $this->objFromFixture(Member::class, 'admin_user');
+        $this->logInAs($admin);
+
+        // Create new ChangeSet object
+        $changeset = ChangeSet::create();
+        $changeset->ID = '123';
+        $changeset->Name = 'changeset 123';
+        $changeset->State = 'open';
+        $changeset->IsInferred = false;
+        
+        $changeset->write();
+
+        // ChangeSet should be accessable for Admin
+        $this->assertTrue($changeset->canView());
+
+        $expectedStatus = [
+            '200' => [
+                'ID' => '123',
+                'Name' => 'show'
```

Comment:
Looking at the original method, I really don't understand why we decided to add a `show` parameter there. My instinct would be to remove the show logic from the original method and not test it.

Reply:
DONE